### ISSUE TITLE
tfwallet: TypeError: t.ed25519 is not a function. (In 't.ed25519()', 't.ed25519' is undefined on initializeImportedPkidAccounts

### ISSUE THREAD

**Issue by ken-de-moor** [threefoldtech/threefold_wallet#245]
2 : {
timestamp : Wed, 26 May 2021 08:17:02 GMT,
message : initializeSingleAccount,
ctx : {
index : 0,
revine : 013bbc20a741e96220c3fe56e5988bb98066bc0646bb08018583b88245184909ae8011fea53aae,
stellar : GDUJCCM3AFD4K3QLJ53TJJCWPJOH4S22CQIABLUX2WDI3FV6JIZJVW3R,
type : imported,
},
level : INFO,
},
3 : {
timestamp : Wed, 26 May 2021 08:17:02 GMT,
message : error initializeImportedPkidAccounts,
ctx : {
error : TypeError: t.ed25519 is not a function. (In 't.ed25519()', 't.ed25519' is undefined),
},
level : ERROR,
},
4 : {
timestamp : Wed, 26 May 2021 08:17:06 GMT,
message : fetchedAccount,
ctx : {
revineAddress : 013bbc20a741e96220c3fe56e5988bb98066bc0646bb08018583b88245184909ae8011fea53aae,
stellarPubKey : GDUJCCM3AFD4K3QLJ53TJJCWPJOH4S22CQIABLUX2WDI3FV6JIZJVW3R,
},
level : INFO,
},
]

**Comment by ken-de-moor** [comment]
Fix live on staging, doing a regression before prod deploy since it has potential impact on other flows.

**Comment by abdulgig** [comment]
@ken-de-moor , just to let you know that the user is still facing the same problem. We have requested for the logs again

![image](https://user-images.githubusercontent.com/25789764/120916208-4d1e1a80-c6b9-11eb-8b8c-c364c503d2c9.png)

**Comment by abdulgig** [comment]
Logs shared by the user,

```
[
0 : {
timestamp : Sun, 06 Jun 2021 07:38:42 GMT,
message : startedWallet,
ctx : [VAR: UNDEFINED],
level : INFO,
},
1 : {
timestamp : Sun, 06 Jun 2021 07:38:42 GMT,
message : initialize,
ctx : [VAR: UNDEFINED],
level : INFO,
},
2 : {
timestamp : Sun, 06 Jun 2021 07:38:43 GMT,
message : initializeSingleAccount,
ctx : {
index : 0,
revine : 013bbc20a741e96220c3fe56e5988bb98066bc0646bb08018583b88245184909ae8011fea53aae,
stellar : GDUJCCM3AFD4K3QLJ53TJJCWPJOH4S22CQIABLUX2WDI3FV6JIZJVW3R,
type : imported,
},
level : INFO,
},
3 : {
timestamp : Sun, 06 Jun 2021 07:38:43 GMT,
message : error initializeImportedPkidAccounts,
ctx : {
error : TypeError: t.ed25519 is not a function. (In 't.ed25519()', 't.ed25519' is undefined),
},
level : ERROR,
},
4 : {
timestamp : Sun, 06 Jun 2021 07:38:44 GMT,
message : fetchedAccount,
ctx : {
revineAddress : 013bbc20a741e96220c3fe56e5988bb98066bc0646bb08018583b88245184909ae8011fea53aae,
stellarPubKey : GDUJCCM3AFD4K3QLJ53TJJCWPJOH4S22CQIABLUX2WDI3FV6JIZJVW3R,
},
level : INFO,
},
]```

**Comment by ken-de-moor** [comment]
@abdulgig another fix is live.

**Comment by JonasWijne** [comment]
should be fixed on prod version v2.8.1, successfully tested it myself with a devseed on staging enviroment
